core/pom.xml

```diff
@@ -32,6 +32,10 @@ under the License.
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
```

Comment:
i add it in an other pull request to core, so i added it here too, otherwise it would not compile.